pom.xml

```diff
@@ -26,8 +26,9 @@
 
     <developers>
         <developer>
-            <id>slide_o_mix</id>
+            <id>slide</id>
```

Comment:
I think this is supposed to match the Jenkins infra username, not the GitHub username. So, in this case, my Jenkins username is slide_o_mix.